Library/Formula/ola.rb

```diff
@@ -1,8 +1,8 @@
 class Ola < Formula
   desc "Open Lighting Architecture for lighting control information"
   homepage "https://www.openlighting.org/ola/"
-  url "https://github.com/OpenLightingProject/ola/releases/download/0.9.8/ola-0.9.8.tar.gz"
-  sha256 "1c1e0fc1810b0c0857563bc481c872b8ed5d2e62c97c1083c82eabd7ebbd70a6"
+  url "https://github.com/OpenLightingProject/ola/releases/download/0.10.0/ola-0.10.0.tar.gz"
+  sha256 "cae8131a62f0ff78d399c42e64a51b610d7cf090b606704081ec9dd2cf979883"
   revision 1
```

Comment:
Yeah, please.

> but that has been there since before I started contributing updates to this formula, and I don’t know its semantic purpose.

Apologies. `revision`s are our way of mandating people reinstall the formula locally for a particular reason.

Sometimes between upstream releases on formulae things like security problems, compatibility problems against a newer version of something it needs to function, unexpected runtime breakage, etc crop up that require a rebuild/reinstall to fix.

Since there's no new upstream release available in 90% of those cases we add a `revision` line to increase the formula's version which then enforces that upgrade for end users. When there's a new upstream release, the `revision` is reset to `0` which is unwritten so we just kill the line.